channelmonitor/channelmonitor_test.go

```diff
@@ -550,7 +558,7 @@ func TestChannelMonitorTimeouts(t *testing.T) {
 					mch := m.AddPushChannel(ch1).(*monitoredPushChannel)
 					chCtx = mch.ctx
 				} else {
-					mch := m.AddPushChannel(ch1).(*monitoredPushChannel)
```

Comment:
How did that work before?

Reply:
It didn't 😆 
It tests the right thing now and still works so that's good news 🎉